### ISSUE TITLE
Icon image doesn't work

### ISSUE THREAD

**Issue by nandorojo** [dominicstop/react-native-ios-context-menu#25]
When I set `icon.iconType` to `REQUIRE`, I see this issue:

<img width="300" alt="Screen Shot 2021-11-10 at 1 40 19 PM" src="https://user-images.githubusercontent.com/13172299/141173733-7a95a2b5-b41a-4fd2-af43-03ac39fb735e.png">

It's on a simulator using iOS 14.5.

I opened the url of that image in the Safari, and it works there:

<img width="300" alt="Screen Shot 2021-11-10 at 1 42 36 PM" src="https://user-images.githubusercontent.com/13172299/141173883-6a0802ed-76d7-41ae-8445-3e12d8af9d6b.png">

Everything else about this library works _amazingly_, I absolutely love it! This is the only issue I've had so far.

This is what my code looks like for setting the icon:

```tsx
import { Image } from 'react-native'
const iconValue = Image.resolveAssetSource(require('./camera-outline.png'))

console.log({ iconValue })
const icon = {
  iconType: 'REQUIRE',
  iconValue
}
```

That logs this, so I know it's getting the source properly:

<img width="929" alt="Screen Shot 2021-11-10 at 1 44 39 PM" src="https://user-images.githubusercontent.com/13172299/141174153-e020ea26-2137-4606-93cf-38534fac865e.png">

**Comment by dominicstop** [comment]
Hey this has been fixed in version `1.5.0` (so i'll be closing this issue now...) ✨

See [ContextMenuViewExample18](https://github.com/dominicstop/react-native-ios-context-menu/blob/v1.6.0/example/src/examples/ContextMenuViewExample18.tsx) (or the example snippet below) for some example usage on how configure a menu action to use to use a custom image for its icon 😌

https://user-images.githubusercontent.com/18517029/143518873-5a96577c-7ef8-47b9-91fa-b0bd11410136.mov

![Screen Shot 2021-11-26 at 10 44 40 AM](https://user-images.githubusercontent.com/18517029/143518948-2d0d7296-5fac-4615-8dc8-c95848397407.png)

<br>

## Example Snippet
```jsx
import * as React from 'react';
import { Image } from 'react-native';

import { ContextMenuView } from 'react-native-ios-context-menu';

const imageIcon = Image.resolveAssetSource(
  require('../assets/emoji-sparkling-heart.png')
);

export function Example(props) {
  return (
    <ContextMenuView
      menuConfig={{
        menuTitle: 'ContextMenuViewSimpleExample18',
        menuItems: [{
          actionKey  : 'key-02'   ,
          actionTitle: 'Action #2',
          discoverabilityTitle: 'Use "IMAGE_REQUIRE" icon',
          icon: {
            type : 'IMAGE_REQUIRE',
            imageValue: imageIcon,
          }
        }],
      }}
    >
      {/** ... */}
    </ContextMenuView>
  );
};
```
<br>
Thanks 💖

**Comment by nandorojo** [comment]
You’re on a roll 🔥
